android/src/main/java/com/pdftron/pdftronflutter/helpers/PluginUtils.java

```diff
@@ -856,6 +863,18 @@ public static ConfigInfo handleOpenDocument(@NonNull ViewerConfig.Builder builde
                        }
                    }
                 }
+                if (!configJson.isNull(KEY_CONFIG_DEFAULT_ERASER_TYPE)) {
+                    String eraserType = configJson.getString(KEY_CONFIG_DEFAULT_ERASER_TYPE);
+                    if (DEFAULT_ERASER_TYPE_ANNOTATION.equals(eraserType)) {
+                        toolManagerBuilder = toolManagerBuilder.setEraserType(Eraser.EraserType.ANNOTATION_ERASER);
+                    }
+                    if (DEFAULT_ERASER_TYPE_HYBRID.equals(eraserType)) {
```

Comment:
should be else if as there should be only 1 possibility

Reply:
I have changed it to an if-else statement